### DISCUSSION TITLE
Firebase Dynamic Link Always Opens App Store

### DISCUSSION THREAD

**Discussion by batuhanoztrk** [invertase/react-native-firebase#6570] [Q&A]
Was trying to setup deep-linking with Firebase Dynamic Link but the link always direct the user back to App Store even though the app has been installed.

I have tried the "Link Preview (Debug)" to test my Dynamic Link. Clicking the "App with deep link" and "Use Custom Scheme" launch the application properly, and routed the user to the correct page within the app. So I would assumed that my AASA and application are setup properly.

I have verified many times on the iOS bundle ID, App Store ID and Team ID configured within Firebase. All of them are populated properly, but still the link will always direct me to the App Store. Anyone can advise what else might caused this problem?

**Comment by mikehardy** [comment]
It may be that the applinks file has not been downloaded on device yet (ios), or that you have not registered the protocol / custom domain correctly in android manifest / ios plist, that's all I can think of at the moment

**Reply by batuhanoztrk** [reply]
When I enter the address xxxx.web.app/content/89, which I specified as a deep link, the application redirects without any problems. But when I enter the address xxxx.page.link/xxxx, if the application is installed on Android without any problems, it is directed to the store if it is not installed on the deep link. On Ios, it always leads to the store. Added "applinks:xxxx.web.app" as applinks in Info.plist. If I add the address xxxx.page.link, this time I cannot provide the correct redirect within the application. The app just opens.
